### ISSUE TITLE
Add TEX source for report to repository

### ISSUE THREAD

**Issue by jmshapir** [jmshapir/TestWorkflow#4]
Please add the TEX source file for your report to 

/source/drafts/report.tex

and update the file locations to use relative paths to the plots in output (../../output/analysis/XX.pdf) so that the file compiles.

Please compile and put the PDF in /output/drafts/report.pdf, being sure to track it with git-lfs.

**Comment by francescomariaesposito** [comment]
@jmshapir I am reporting here, step by step, what I have done for this issue. Since this is the first time I am using this, do you mind double checking that everything is ok? Please let me know if I should have skipped and/or added some steps and/or if they are all in the right order.
Please notice also that above you mentioned that the path for pictures should be ../../output/analysis/XX.pdf. I have used instead ../../output_large/analysis/XX.pdf. According to Wiki/Repository Structure it seems that pictures should be located in output_large and not in output, but of course I may be wrong. Also all other figures were already in output_large.
Finally, you did not specify which report you wanted. So, for the sake of completeness, I have uploaded all the three reports I have written so far. The first one (9th of July) is .doc so I uploaded it directly in output/drafts/ , while the other two reports (20th and 28th of July) are .tex and so their respective codes are in source/drafts/ and their .pdf are in output/drafts/. Of course in the .tex files I have changed all the relative paths for including graphs as ../../output_large/analysis/XX.pdf.

So the steps I have followed are, in order, the following:
1. cd C:\Users\fesposit\Desktop\GitHub
2. git clone https://github.com/jmshapir/TestWorkflow.git C:\Users\fesposit\Desktop\GitHub\TestWorkflow
3. git add -A
4. On my local repository, I have created all needed folders when they where not already there and then I uploaded all .pdf figures in output_large/analysis/ ; all .tex report in source/drafts/ and all .pdf reports in output/drafts
5. git lfs install
6. cd C:\Users\fesposit\Desktop\GitHub\TestWorkflow
7. git lfs track output/drafts/\* (so to track also the .doc report)
8. git lfs track output_large/analysis/\* (to be sure of tracking also all .pdf figures). These 2 commands (7. & 8.) just modified  .gattributes. They added two more lines to it, i.e. respectively output/drafts/\* filter=lfs diff=lfs merge=lfs -text & output_large/analysis/\* filter=lfs diff=lfs merge=lfs -text.
9. git add .gitattributes
10. git add source\drafts
11. git add output\drafts
12. git add output_large\analysis
13. git commit -m "Uploaded figures, reports .tex, reports .pdf and a report .doc for #4"
14. git remote rm origin
15. git remote add origin https://github.com/jmshapir/TestWorkflow.git
16. git push -u origin master

Please let me know if everything is fine for you. Then, in case, please feel free to close the issue.

**Comment by jmshapir** [comment]
Thanks @francescomariaesposito. The only report I want is the final report related to the Ethnicity project.

**Comment by jmshapir** [comment]
Please remove the other reports and any other files that you uploaded related to the protest project.

**Comment by francescomariaesposito** [comment]
@jmshapir I am sorry for the misunderstanding. I have just done what you requested. I have:
1. Removed all files related to the protest project.
2. Uploaded all files, in the folders specified by you, related to the ethnicity project (figures, report.tex and report.pdf). 
3. Pushed all my changes on the master branch.

Please just notice that among the other figures, in output_large/analysis, there is also a tex file "desc_stat_tab_3" which is a tex code for a table that I call from the report.tex so that the report compiles correctly.

Everything should also be tracked with git-lfs as you requested.
Please if you are fine with this issue, close it.
Thanks

**Comment by jmshapir** [comment]
@francescomariaesposito my intention was for you to only upload report.text and report.pdf--I expected the figures we need would already be present because I pulled them from the GoogleDrive folder. Can you clarify?

**Comment by francescomariaesposito** [comment]
@jmshapir Yes you are totally right. All figures were already included in the Google Drive folder. Indeed, I did not uploaded any figure here. I just uploaded Report.tex (modifying relative paths) and Report.pdf (after compile). Moreover, I uploaded desc_stat_tab_3.tex in output_large/analysis (the floder where all the pictures are). This is because report.tex calls for the table desc_stat_tab_3.tex (which is a stata output created by analysis.do).
